packages/plugin-doc-coverage/src/lib/runner/doc-processer.unit.test.ts

```diff
@@ -0,0 +1,278 @@
+import type { ClassDeclaration, VariableStatement } from 'ts-morph';
+import {
+  nodeMock,
+  sourceFileMock,
+} from '../../../mocks/source-files-mock.generator';
+import {
+  getClassNodes,
+  getUnprocessedCoverageReport,
+  getVariablesInformation,
+  mergeCoverageResults,
+} from './doc-processer.js';
+import type { CoverageReportShape } from './models.js';
+
+describe('getUnprocessedCoverageReport', () => {
```

Comment:
As we renamed the types I suggest to also rename this function.

e.g.: `getDocumentationReport`